proto/dydxprotocol/clob/query.proto

```diff
@@ -198,6 +198,15 @@ message StreamUpdate {
   }
 }
 
+// StagedFinalizeBlockEvent is an event staged during `FinalizeBlock`.
```

Comment:
Can we add context to this comment to explain staging more and why it is necessary? + this is not exposed externally, so maybe move it out of query.proto?

Reply:
Will add more documentation in code as opposed to proto file